src/components/Utils/formFieldValidators.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { get } from 'lodash';
+
+export const combineValidators = (validators) => (...args) => validators
+  .map(item => item(...args))
+  .find(item => item);
+
+const reduceLocations = (locations, propName) => {
```

Comment:
name doesn't say a lot, what result of the reduce? it can be anything

Reply:
I see you just moved code, but please rename

Reply:
@NikitaSedyx  criticize - offer )

Reply:
oximiron?)

I assume it's `getAllLocationsCount`?